vision_agent_tools/tools/shared_types.py

```diff
@@ -1,8 +1,15 @@
+from enum import Enum
 from pydantic import BaseModel
 
 
+class Device(str, Enum):
+    GPU = "cuda"
+    CPU = "cpu"
```

Comment:
@hrnn @dillonalaird @camiloaz Should we add `mps` for mac users?

Reply:
Fixed